modules/nf-core/hiphase/tests/main.nf.test

```diff
@@ -0,0 +1,97 @@
+nextflow_process {
+    name "Test Process HIPHASE"
+    script "../main.nf"
+    process "HIPHASE"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "hiphase"
+
+    test("hiphase - vcf") {
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/pacbio/vcf/NA03697B2_new.pbmm2.repeats.vcf.gz', checkIfExists: true),
+                ]
+
+                input[1] = [
+                    [ id:'test' ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/pacbio/vcf/NA03697B2_new.pbmm2.repeats.vcf.gz.csi', checkIfExists: true),
+                ]
+
+                input[2] = [
+                    [ id:'test' ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/pacbio/bam/NA03697B2_downsampled.pbmm2.repeats.bam', checkIfExists: true),
+                ]
+
+                input[3] = [
+                    [ id:'test' ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/pacbio/bam/NA03697B2_downsampled.pbmm2.repeats.bam.bai', checkIfExists: true),
+                ]
+
+                input[4] = [
+                    [ id:'test' ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/genome3.fasta', checkIfExists: true),
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert file(process.out.vcf[0][1]).name == "test.phased.vcf" },
```

Comment:
same check if the nft-vcf plugin can be used here

Reply:
I now verify the snapshot of the vcf output which should be more comprehensive

Reply:
ended up just verifying the snapshot of the whole output - hopefully that is comprehensive enough?